frontend/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,7 @@
+export default function HomePage() {
+    return (
```

Comment:
What does this page do? Do you plan on adding content here too or is this only here for the general formatting?

Reply:
I had a plan but then forget about it and as it seems also forgot to delete this unused page :) thank you for noticing!